packages/components/src/components/primitives/PinInput/PinInput.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { Fragment } from 'react';
+import { Meta } from '@storybook/react';
+import { PinInput } from './PinInput';
+import { Table, Tbody, Td, Th, Thead, Tr } from '@chakra-ui/react';
+import { pinInputSizes, pinInputVariants } from './types';
+
+export default {
+  component: PinInput,
+  title: 'Components/Primitives/PinInput',
+  argTypes: {
+    variant: {
+      options: pinInputVariants,
+      control: { type: 'select' },
+    },
+    size: {
+      options: pinInputSizes,
+      control: { type: 'select' },
+    },
+  },
+} as Meta;
+
+const states = ['default', 'filled', 'error', 'disabled', 'disabled-and-filled'];
+
+export const All = () => (
```

Comment:
I think we are not doing anything with the `error` state. Could we set `isInvalid` to `true` when error state? @TheEnmanuel23

Reply:
Fixed, good catch!